x/emissions/module/rewards/rewards_test.go

```diff
@@ -270,6 +273,32 @@ func (s *RewardsTestSuite) TestStandardRewardEmission() {
 	})
 	s.Require().NoError(err)
 
+	scoresAtBlock, err := s.emissionsKeeper.GetReputersScoresAtBlock(s.ctx, topicId, block)
+	s.Require().Equal(len(scoresAtBlock.Scores) <= int(params.GetMaxTopReputersToReward()), true, "Only few Top reputers can get reward")
+
+	networkLossBundles, err := s.emissionsKeeper.GetNetworkLossBundleAtBlock(s.ctx, topicId, block)
+	s.Require().NoError(err)
+
+	inferers, _, err := rewards.GetInferenceTaskRewardFractions(
+		s.ctx,
+		s.emissionsKeeper,
+		topicId,
+		block,
+		params.PRewardSpread,
+		networkLossBundles,
+	)
+
+	forecasts, _, err := rewards.GetForecastingTaskRewardFractions(
+		s.ctx,
+		s.emissionsKeeper,
+		topicId,
+		block,
+		params.PRewardSpread,
+		networkLossBundles,
+	)
+	s.Require().Equal(len(inferers) <= int(params.GetMaxTopWorkersToReward()), true, "Only few Top workers can get reward")
+	s.Require().Equal(len(forecasts) <= int(params.GetMaxTopWorkersToReward()), true, "Only few Top workers can get reward")
+
```

Comment:
Not sure if it's a good idea to reuse this test for the following reasons:

1. It's not the main purpose of the test.
2. The default initial value for `MaxTopWorkers` and `MaxTopReputers` is 20. This test is only considering 5 actors for both, so it's not actually testing anything.

Reply:
Yes, `MaxTopWorkers` and `MaxTopReputers` is 20. So I did a test by changing the values as 5 and 4. And reverted it when commmit. Then I will make the script with more than 20 actors. Do you think it does make sense?